html/js/product-multilingual.js

```diff
@@ -704,24 +714,26 @@ function get_recents(tagfield) {
 
                 var $this = $(this);
                 var id = $this.attr('id');
+                var clas= $this.attr("data-info");
```

Comment:
the variable is named "clas" because "class" is a protected word? As the attribute we read is not the class anyway, could we rename the variable to data_info maybe?